routes/team.js

```diff
@@ -28,4 +29,15 @@ team.post("/data", function(req, res) {
 	});
 });
 
-module.exports = team;
\ No newline at end of file
+team.post("/delete", authHelper.isAdmin, function(req, res) {
```

Comment:
@rvanderheyde Would you prefer DELETEing to /users or add some additional path? For example should it be team.delete("/", ...) or team.delete("/user", ...)?